mobile/android/app/build.gradle

```diff
@@ -40,21 +46,35 @@ android {
     }
 
     defaultConfig {
-        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.radioaktywne"
-        // You can update the following values to match your application needs.
-        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
+        applicationId "pl.radioaktywne"
```

Comment:
Did you consult this with @lewandowski-jan? A package name change could be crucial when it comes to publishing the app